uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-day.component.spec.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'leave-absences/shared/components/leave-calendar-day.component'
+], function () {
+  'use strict';
+
+  describe('leaveCalendarDay', function () {
+    var $componentController, $log;
+
+    beforeEach(module('leave-absences.templates', 'leave-absences.components'));
+    beforeEach(inject(function (_$componentController_, _$log_) {
```

Comment:
Honestly i'm still on the fence about this, i don't know why sometimes we have separate lines between the various beforeEach at the beginning